lib/monitoring/ci-alarms.ts

```diff
@@ -44,12 +44,13 @@ export class JenkinsMonitoring {
       treatMissingData: TreatMissingData.BREACHING,
     }));
 
-    this.alarms.push(new Alarm(stack, 'MainNodeTooManyJenkinsProcessesFound', {
-      alarmDescription: 'Only one jenkins process should run at any given time on the main node, there might be a cloudwatch configuration issue',
-      metric: mainNode.ec2InstanceMetrics.foundJenkinsProcessCount.with({ statistic: 'max' }),
-      evaluationPeriods: 3,
+    this.alarms.push(new Alarm(stack, 'MainNodeJenkinsProcessNotFound', {
+      alarmDescription: 'Jenkins process is not running',
+      metric: mainNode.ec2InstanceMetrics.foundJenkinsProcessCount.with({ statistic: 'avg' }),
```

Comment:
is `avg` the right stat? Should we change it to `sum` to make sure at any point of time total number of process should not be less than sum of all the process count at that data point, which should always be 1

Reply:
@rishabh6788 thanks I believe sum might not work here, sometimes the value could be 0.89, the average to check to for 1 is what is right, as we should only check 1 or 0 up or down and trigger an alarm.

Reply:
I have also tested the alarm , to stop and start the jenkins container works as expected